ext/tls/tls_key.rs

```diff
@@ -222,6 +224,8 @@ pub struct TlsKeyLookup {
     RefCell<HashMap<String, broadcast::Sender<Result<TlsKey, ErrorType>>>>,
 }
 
+impl deno_core::GcResource for TlsKeyLookup {}
```

Comment:
Q: is it mandatory to implement GcResource for structs that are marked as cppgc to be GCed? What's the role of GcResource? We have some ops that are marked cppgc but the return types don't implement GcResource.

Reply:
Yes, its mandatory to be explicit. This is to prevent accidentally wrapping the wrong types:

```rust
#[op2]
#[cppgc]
// previously this could end up wrapping `Option<Thing>` as 
// a cppgc object
fn op_something() -> Option<Thing> {} 
```

Reply:
Okay. 

Looks like all cppgc ops are updated in this PR.